Library/Formula/spiff.rb

```diff
@@ -0,0 +1,15 @@
+class Spiff < Formula
+  desc "Tool and declarative YAML templating system specially for BOSH manifests."
+  homepage "https://github.com/cloudfoundry-incubator/spiff"
+  url "https://github.com/cloudfoundry-incubator/spiff/releases/download/v1.0.7/spiff_darwin_amd64.zip"
```

Comment:
According to https://github.com/cloudfoundry-incubator/spiff/tree/master#installation
> ... the only supported way to use spiff is to use an official binary release.
Would you still recommend to build it from source? Thanks.